piccolo/query/methods/alter.py

```diff
@@ -36,6 +36,21 @@ def ddl(self) -> str:
         return f"RENAME TO {self.new_name}"
 
 
+@dataclass
+class RenamePkConstraint(AlterStatement):
+    __slots__ = ("old_constraint_name", "new_constraint_name")
+
+    old_constraint_name: str
+    new_constraint_name: str
```

Comment:
Lets change these to `old_name` and `new_name`. I don't think we need to be any more explicit than that.